### DISCUSSION TITLE
CEMU not recognized ToyPad Emulator

### DISCUSSION THREAD

**Discussion by Lohusss** [Berny23/LD-ToyPad-Emulator#112] [Q&A] [answered]
![Снимок экрана (2)](https://user-images.githubusercontent.com/126674862/222151660-ede9bbd2-2d7c-42a3-936c-52dec193dc25.png)
Worked great for 3 days. I'm guessing it's because of the LEGO READER V2.10 driver update, but after rolling back the update, the problem persisted... Can anyone help?

**Comment by Lohusss** [comment]
Did everything in turn according to the manual and more than once. "Troubleshooting" didn't help much either.

**Comment by Lohusss** [comment, marked as answer]
I found a solution, I'm a fool, you just need to manually roll back the LEGO READER driver to the usb input device

**Reply by Cloudy1011** [reply]
how do you do that?
